job-dsls/gradlew

```diff
@@ -28,16 +44,16 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
just to assure this double quote thing is ok.... it looks weird and I'm wondering whether it should be `'-Xmx64m -Xms64m'` instead

Reply:
isn't this change from updated gradlew script, i guess @mbiarnes didn't this manually

Reply:
@Ginxo nop - this was automatic - looks strange - but it works